### ISSUE TITLE
Prompt for FTP username/password

### ISSUE THREAD

**Issue by BugsBunny338** [dg/ftp-deployment#125]
**TL;DR**
`$ php deployment.phar deployment.ini --username=bugsbunny --password=abc123`

**Description**
I wish I would be prompted for FTP username/password, or at least to be able to specify it via param, something like `$ php deployment.phar deployment.ini --username=bugsbunny --password=abc123`.

**Reasoning**
The `deployment.ini` file is different for every project and sometimes its configuration takes non-trivial amount of time to build. Therefore I'd like the `deployment.ini` part of the git repo. However, I feel very uncomfortable pushing a plain-text password to a git repository, even to a private one.

**Comment by BugsBunny338** [comment]
Ah, just found https://github.com/dg/ftp-deployment/issues/84 and https://github.com/dg/ftp-deployment/issues/75, never mind 😞
